src/Pages/URLShortener/URLShortener.js

```diff
@@ -66,19 +66,19 @@ export default function URLShortenerPage(props) {
     }
   }
 
+  async function maybeSubmitUrlOnKeyPresses(e) {
+    if ((e.ctrlKey || e.metaKey) && (e.key === 'Enter')) {
+      maybeSubmitUrl(url);
```

Comment:
Maybe make the resetting of fields in this function, not in the function this references (Code orthogonality)